src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+  const newDateFormat = [];
+  const separator = toFormat[3];
+
+  const year = fromFormat.find(el => el.includes('Y'));
+
+  toFormat.forEach(newElement => {
```

Comment:
You could potentially change the name of the newElements it's kinda confusing. Maybe something like `newFormatElement` also you could potentially use map and return values instead of pushing them to an array outside of the scope of this method. This however would force you to pop the last element, separator or undefined value depending on you implementation. Nothing wrong with forEach here, good implementation of the formatDate overall.